bom/pom.xml

```diff
@@ -241,7 +241,7 @@ THE SOFTWARE.
       <dependency>
         <groupId>commons-beanutils</groupId>
         <artifactId>commons-beanutils</artifactId>
-        <version>1.9.3</version>
+        <version>1.9.4</version>
```

Comment:
```suggestion
        <version>1.9.3</version>
```

Reply:
I am interested by the PR, our security AppScan are not happy about this commons-beanutils 3rd party.

https://vuln.whitesourcesoftware.com/vulnerability/CVE-2014-0114
which is fixed on version 1.9.4

Reply:
If you can help fix the failing tests caused by the upgrade, we can upgrade.

Reply:
And that was also caused by an upgrade of the library. We know doing that breaks things, that's the whole problem.